src/main/java/org/broadinstitute/hellbender/tools/exome/CreateAllelicPanelOfNormals.java

```diff
@@ -0,0 +1,94 @@
+import org.broadinstitute.hellbender.tools.pon.allelic.AllelicPanelOfNormalsCreator;
+import org.broadinstitute.hellbender.utils.Utils;
+import org.broadinstitute.hellbender.utils.hdf5.HDF5File;
+import org.broadinstitute.hellbender.utils.text.XReadLines;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Create an allelic panel of of normals (for use in correcting reference bias in the allele-fraction model),
+ * given het pulldowns for samples that are part of the panel.  Output to both HDF5 and TSV files is supported.
+ *
+ * @author Samuel Lee &lt;slee@broadinstitute.org&gt;
+ */
+@CommandLineProgramProperties(
+        summary = "Creates an allelic panel of normals, given het pulldowns for samples that are part of the panel, " +
+                "and writes it to an HDF5 file (and optionally, a TSV file).  Samples can differ from those " +
+                "used to create the coverage PoN but should be representative of the same sequencing process.",
+        oneLineSummary = "Creates an allelic panel of normals",
+        programGroup = CopyNumberProgramGroup.class
+)
+public final class CreateAllelicPanelOfNormals extends CommandLineProgram {
+    protected static final String SITE_FREQUENCY_SHORT_NAME = "f";
+    protected static final String SITE_FREQUENCY_LONG_NAME = "siteFrequency";
+
+    protected static final String TSV_OUTPUT_FILE_SHORT_NAME = "tsvOut";
+    protected static final String TSV_OUTPUT_FILE_LONG_NAME = "tsvOutput";
+
+    @Argument(
+            doc = "Input pulldown files for all samples in the panel of normals.  " +
+                    "Duplicate samples are not removed.",
+            shortName = StandardArgumentDefinitions.INPUT_SHORT_NAME,
+            fullName = StandardArgumentDefinitions.INPUT_LONG_NAME,
+            optional = false
+    )
+    protected List<File> inputFiles = new ArrayList<>();
+
+    @Argument(
+            doc = "Output HDF5 file.  The allelic panel of normals will be written to this file (which will be overwritten, if it exists).",
+            shortName = StandardArgumentDefinitions.OUTPUT_SHORT_NAME,
+            fullName = StandardArgumentDefinitions.OUTPUT_LONG_NAME,
+            optional = false
+    )
+    protected File outputFile;
+
+    @Argument(
+            doc = "Output TSV file.  The allelic panel of normals will be written as tab-separated values to this file.",
+            shortName = TSV_OUTPUT_FILE_SHORT_NAME,
+            fullName = TSV_OUTPUT_FILE_LONG_NAME,
+            optional = true
+    )
+    protected File outputTSVFile = null;
+
+    @Argument(
+            doc = "Site frequency threshold.  Sites that appear in a fraction of the samples that is strictly less than this threshold are not included in the panel.",
+            shortName = SITE_FREQUENCY_SHORT_NAME,
+            fullName = SITE_FREQUENCY_LONG_NAME,
+            optional = true
+    )
+    protected double siteFrequency = 0.25;
```

Comment:
Would it be more clear to make it something like `minSiteFrequency`, or `siteFrequencyThreshold`